2025-05-26 17:19:40 | ERROR    | skymapper.main:main:162 - Error in main: process_image_cli() missing 3 required positional arguments: 'image_path', 'calibration_file', and 'output'
2025-05-26 17:22:58 | ERROR    | skymapper.main:main:201 - Error in main: process_image_cli() missing 3 required positional arguments: 'image_path', 'calibration_file', and 'output'
2025-05-26 17:28:59 | INFO     | skymapper.main:calibrate:192 - Generating calibration from 1 images
2025-05-26 17:28:59 | INFO     | skymapper.calibration:generate_calibration_from_stars:337 - Extracting patches from image: /home/vberenz/Workspaces/skymap/images/
2025-05-26 17:28:59 | WARNING  | skymapper.calibration:generate_calibration_from_stars:340 - Could not load image: /home/vberenz/Workspaces/skymap/images/
2025-05-26 17:28:59 | ERROR    | skymapper.calibration:generate_calibration_from_stars:419 - Error generating calibration: No suitable patches found in any image
2025-05-26 17:28:59 | ERROR    | skymapper.main:calibrate:204 - Error generating calibration: No suitable patches found in any image
2025-05-26 17:32:29 | ERROR    | skymapper.main:calibrate:213 - Error generating calibration: name 'Path' is not defined
2025-05-26 17:32:46 | INFO     | skymapper.main:calibrate:195 - Found 3 TIFF images in /home/vberenz/Workspaces/skymap/images/
2025-05-26 17:32:46 | INFO     | skymapper.main:calibrate:204 - Generating calibration from 3 images
2025-05-26 17:32:46 | INFO     | skymapper.calibration:generate_calibration_from_stars:337 - Extracting patches from image: /home/vberenz/Workspaces/skymap/images/nightskycam3_2025_04_07_04_21_30.tiff
2025-05-26 17:32:46 | INFO     | skymapper.calibration:generate_calibration_from_stars:337 - Extracting patches from image: /home/vberenz/Workspaces/skymap/images/nightskycam3_2025_04_29_00_07_30.tiff
2025-05-26 17:32:46 | INFO     | skymapper.calibration:generate_calibration_from_stars:337 - Extracting patches from image: /home/vberenz/Workspaces/skymap/images/nightskycam3_2025_04_05_03_54_30.tiff
2025-05-26 17:32:47 | ERROR    | skymapper.calibration:generate_calibration_from_stars:419 - Error generating calibration: No suitable patches found in any image
2025-05-26 17:32:47 | ERROR    | skymapper.main:calibrate:216 - Error generating calibration: No suitable patches found in any image
